apps/mobile/src/app/assets/index.ts

```diff
@@ -2,5 +2,6 @@ import logo from './images/logo.png';
 import textLogo from './images/text-logo.png';
 import textLogoInverted from './images/text-logo-inverted.png';
 import placeholder from './images/placeholder.png';
+import timeOutline from './images/time-outline.png';
```

Comment:
I suggest to use an icon that already exists and definitelly an SVG instead of PNG. Check Icon component in atoms, we use react-native-vector-icons there and you definitely can found an icon like https://fonts.google.com/icons?selected=Material%20Icons%3Aquery_builder%3A from Material Icons